acceptance/config/aio/options.rb

```diff
@@ -1,9 +1,3 @@
 {
   :type => 'aio',
```

Comment:
Are these really even needed? The type appears to be implied by the file path, which the beaker-puppet changes appear to know how to find.

Reply:
They are mostly not needed, since the CI type above is typically the same as the rake task name, e.g. `ci:test:aio`. However, that's not true for the `ci:aio:gem` task (in puppet and eventually in facter), as it relies on the `git` CI type. I think having it explicit makes it cleaner, though more verbose.